appveyor.yml

```diff
@@ -0,0 +1,24 @@
+# Configuration file for AppVeyor continuous integration on Windows
+# https://ci.appveyor.com/project/josef-pkt/statsmodels
+
+#os: Windows Server 2012
+
+platform:
+  - x64
+
+install:
+  - ps: Invoke-WebRequest "https://bootstrap.pypa.io/ez_setup.py" -OutFile "c:/ez_setup.py"
+  - ps: Invoke-WebRequest "https://bootstrap.pypa.io/get-pip.py" -OutFile "c:/get-pip.py"
+  - C:\Python27\python.exe c:\get-pip.py
+  - C:\Python27\Scripts\pip.exe install conda
+  - C:\Python27\Scripts\conda.exe init
+  - C:\Python27\Scripts\conda.exe create -n py3 --yes --quiet python=3 numpy scipy cython pandas pip nose patsy
+
+build_script:
+  - C:\Python27\envs\py3\python.exe setup.py install
+
+test_script:
+  - cd C:\Python27\envs\py3\Scripts
```

Comment:
> I can try this for the next test run.
> Is there an activate.bat in the main python scripts folder? without conda I only have it in the virutal env scripts folder

Per its setup.py file, conda installs both `activate.bat` and `deactivate.bat` when it is installed.  The default location seems to be the `Scripts` .